package.json

```diff
@@ -175,5 +175,8 @@
     "node": ">= 14.0.0",
     "npm": ">= 6.0.0"
   },
-  "packageManager": "yarn@3.0.2"
+  "packageManager": "yarn@3.0.2",
+  "resolutions": {
+    "d3-color": "~3.1.0"
```

Comment:
what does this do?  I've never seen resolutions before

Reply:
>Yarn supports selective version resolutions, which lets you define custom package versions or ranges inside your dependencies through the resolutions field in your [package.json](https://classic.yarnpkg.com/docs/package-json) file. Normally, this would require manual edits in the yarn.lock file.

Source: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/